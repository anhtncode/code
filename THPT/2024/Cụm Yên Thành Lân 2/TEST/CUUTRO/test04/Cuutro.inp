51 77
34 0 69 24 78 58 62 64 5 45 81 27 61 91 95 42 27 36 91 4 2 53 92 82 21 16 18 95 47 26 71 38 69 12 67 99 35 94 3 11 22 33 73 64 41 11 53 68 47 44 62 
15 36 60
2 41 30
13 17 36
5 7 89
16 26 43
37 47 47
39 41 30
43 49 7
42 45 49
45 50 85
13 35 41
19 35 32
43 43 40
44 48 38
16 34 83
30 44 34
5 44 59
9 27 78
11 44 87
23 40 25
49 49 30
2 42 98
39 49 91
32 48 56
32 32 75
2 49 51
28 41 25
43 49 8
50 50 38
4 24 54
30 44 10
28 34 22
24 47 47
17 25 14
47 47 92
45 50 11
35 43 38
26 34 96
12 26 51
40 43 75
4 41 22
12 45 69
4 13 35
2 41 19
41 41 89
22 25 29
1 49 84
8 21 11
38 49 15
29 33 36
9 29 50
46 47 99
9 17 9
30 47 90
11 46 86
33 48 24
6 25 4
2 16 59
39 39 97
8 24 90
41 50 58
21 33 39
1 39 80
34 43 59
20 26 89
31 42 3
33 41 56
30 38 63
41 46 34
34 48 45
39 44 99
20 40 22
38 46 77
46 50 76
2 5 11
42 46 62
0 39 90
