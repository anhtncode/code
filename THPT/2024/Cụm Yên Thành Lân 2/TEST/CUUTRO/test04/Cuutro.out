198 432 501 567 769 742 746 760 838 878 1087 1153 1264 1259 1323 1396 1336 1300 1387 1411 1448 1517 1581 1618 1413 1457 1276 1322 1310 1436 1484 1582 1618 1590 1482 1441 1364 1515 1678 1494 1639 1490 1484 1301 1085 1112 987 627 473 222 -28 